lib/maestrano/rails/routing/routes.rb

```diff
@@ -3,22 +3,19 @@ class Mapper
     def maestrano_routes
       namespace :maestrano do
         scope module: :rails do
-          get '/metadata', to: 'metadata#index'
           get '/metadata/:tenant', to: 'metadata#index', as: 'tenant'
         end
 
         namespace :auth do
           resources :saml, only:[] do
-            get 'init', on: :collection
```

Comment:
Do we really need retrocompatibility on the routes ? Does it even work as all the controller use params[:tenant] ?